spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe ApplicationController, type: :controller do
+  before(:each) { @user = create(:user) }
+
+  describe "#current_user" do
```

Comment:
Removed Restful methods form describe statement, using only the controller method